source/javascripts/components/_Step.js

```diff
@@ -29,9 +29,31 @@ import { normalizeIconUrl } from "./StepItem/StepItem";
 				return this.id;
 			}
 
-			return this.cvs;
+			return diplayNameFromCvs(this.displayCvs());
 		};
 
+		Step.prototype.displayCvs = function() {
+			return (
+				(this.cvs.startsWith("git::") || this.cvs.startsWith("path::")) 
+					? this.cvs.replace(/(git|path)::/g, "")
+					: this.cvs
+			);
+		}
+
+		Step.prototype.displayTooltip = function() {
+			return `${this.displayName()}<br>${this.displayCvs()}`;
+		}
+
+		function diplayNameFromCvs(cvs) {
+			var lastDelimiter = cvs.lastIndexOf("/");
+
+			if (lastDelimiter != -1) {
+				cvs = cvs.substring(lastDelimiter + 1);
+			}
+
+			return cvs;
+		}
```

Comment:
can you please add unit tests to the new logics in here?

Reply:
Sure, will do.